src/main/java/tools/jackson/databind/ext/datetime/JavaTimeFeature.java

```diff
@@ -0,0 +1,77 @@
+package tools.jackson.databind.ext.datetime;
+
+import tools.jackson.core.util.JacksonFeature;
+
+/**
+ * Configurable on/off features for Java 8 Date/Time module ({@link JavaTimeInitializer}).
+ *
+ * @since 2.16
+ */
+public enum JavaTimeFeature implements JacksonFeature
```

Comment:
If possible, I think we should actually change the idea here to create new `DateTimeFeature` that should ideally control ALL date/time handling (mostly concerning then old `java.util` Date/Calendar and Joda date/time).

It would contain entries we have here, initially, but then see if we could move existing `DeserializationFeature`s/`SerializationFeature`s that are date/time specific into this new enum.

This would then be a building block for

https://github.com/FasterXML/jackson-future-ideas/wiki/JSTEP-5

Finally, renamed `DateTimeFeature` should go in `tools.jackson.databind.cfg` along with existing `EnumFeature` and `JsonNodeFeature`.